shell-switcher.el

```diff
@@ -268,7 +268,7 @@ key is pressed, calls `sswitcher--switch-partially'."
 	 `(lambda () (interactive)
 	    (shell-switcher--switch-partially)
 	    (message ,message)))
-       map) t)
+       map) t (lambda () (switch-to-buffer (current-buffer))))
```

Comment:
could you please explain what this does exactly? Could you please also add a related feature test?